assets/app/view/game/part/icons.rb

```diff
@@ -7,42 +7,88 @@ module View
   module Game
     module Part
       class Icons < Base
+        needs :show_destinations, store: true
```

Comment:
you should probably have a default value here

Reply:
I'm pushing up a commit for this shortly